NetKAN/RN-SovietRockets.netkan

```diff
@@ -23,7 +23,7 @@
     [
 		{ "name"    : "BDAnimationModules" },
 		{ "name" 	  : "ModuleManager" },
-        { "name"    : "TextureReplacer" }
+        { "name"    : "ReflectionPlugin" }
```

Comment:
@raidernick RNSovietRockets depends on the ReflectionPlugin but also conflicts with it (see line 20). Could you fix this?